src/main/java/ch/njol/skript/expressions/ExprListIndex.java

```diff
@@ -0,0 +1,119 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.lang.Variable;
+import ch.njol.skript.lang.util.SimpleExpression;
+import ch.njol.skript.util.LiteralUtils;
+import ch.njol.util.Kleenean;
+import ch.njol.util.coll.CollectionUtils;
+import org.bukkit.event.Event;
+
+import javax.annotation.Nullable;
+import java.util.Map;
+import java.util.TreeMap;
+import java.util.stream.LongStream;
+import java.util.stream.Stream;
+
+@Name("List Index")
+@Description("Returns the index of an object from a list.")
+@Examples({
+	"command /listindex:",
+	"\ttrigger:",
+	"\t\tset {_list::*} to 1,2,3,4,5,1",
+	"\t\tsend \"%first index of 1 in %{_list::*}%%\"",
+	"\t\tsend \"%last index of 1 in %{_list::*}%%\"",
+	"\t\tsend \"%all indexes of 1 in %{_list::*}%%\""
+})
+public class ExprListIndex extends SimpleExpression<Object> {
```

Comment:
Not sure but how would I know when you guys merge this, this isn't version dependant either.

Reply:
> Not sure but how would I know when you guys merge this, this isn't version dependant either.

Generally I believe it's done like `@Since("INSERT VERSION")`, and the version is added when it's merged.